hack/extract-kubeconfig.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+namespace="${1:-}"
+test="${2:-}"
+if [[ -z "${namespace}" || -z "${test}" ]]; then
+	echo "USAGE: $0 <namespace> <test>"
+	exit 1
+fi
+
+function retrieve() {
+	pods="$( oc get pods --namespace ${namespace} --selector "ci.openshift.io/multi-stage-test=${test}" -o json )"
```

Comment:
Why pull this out of a pod, instead of grabbing it from the secret behind `$SHARED_DIR`?

Reply:
You know more about this than me apparently <3